src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests.cs

```diff
@@ -89,7 +90,7 @@ public unsafe partial class A : C, I
             var source = CreateCompilationWithMscorlibAndSystemCore(code);
 
             // the following should not crash
-            source.GetDiagnosticsForSyntaxTree(CompilationStage.Compile, source.SyntaxTrees[0], null, true);
+            source.GetDiagnosticsForSyntaxTree(CompilationStage.Compile, source.SyntaxTrees[0], null, true, includeDiagnosticsWithSourceSuppression: false, cancellationToken: CancellationToken.None);
```

Comment:
minor: You could avoid any changes to this file by making the last two (or three) arguments optional as I suggested above.